### ISSUE TITLE
mss_wms_cl not working

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#48]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/48) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

We don't have MSS_WMSResponse also nothing with a parameter address="COMMAND_LINE"

```
#!python

Traceback (most recent call last):
  File "./mss_wms_cl.py", line 302, in <module>
    main()
  File "./mss_wms_cl.py", line 193, in main
    wsgi = wms.MSS_WMSResponse(mss_wms_settings.nwpaccess, address="COMMAND_LINE")
AttributeError: 'module' object has no attribute 'MSS_WMSResponse'
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

removed not completed command line tool, resolve ticket #48.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #137)

removed not completed command line tool, resolve ticket #48.
